lively.modules/src/system.js

```diff
@@ -291,7 +399,6 @@ function preNormalize (System, name, parent) {
       }
     }
   }
-
   // <snip> experimental
```

Comment:
? :D

Reply:
This is from @rksm, so I will leave this here for the time being.